lua/astrotheme/groups/plugins/avante.lua

```diff
@@ -0,0 +1,13 @@
+---@param c AstroThemePalette
+return function(c)
+  return {
+    AvanteTitle = { fg = c.ui.green, bg = c.ui.base },
+    AvanteReversedTitle = { fg = c.ui.base },
+    AvanteSubtitle = { fg = c.ui.blue, bg = c.ui.base },
+    AvanteReversedSubtitle = { fg = c.ui.base },
+    AvanteThirdTitle = { fg = c.ui.yellow, bg = c.ui.base },
+    AvanteReversedThirdTitle = { fg = c.ui.base },
+    AvanteConflictCurrent = { bg = "#005500" },
+    AvanteConflictIncoming = { bg = "#555500" },
```

Comment:
```suggestion
    AvanteConflictCurrent = "DiffText",
    AvanteConflictIncoming = "DiffAdd",
```

This probably makes the most sense to just link to `DiffText` and `DiffAdd`

Reply:
By default on our end it already links to DiffText and DiffAdd, but you can see it in the issue https://github.com/yetone/avante.nvim/issues/283

Reply:
The main thing is we _do_ set good `DiffAdd` and `DiffText` highlight groups. In this case, we should just remove those highlights and rely on the default

Reply:
Oh wait, I see I see, yeah that isn't good 🤔 we should instead fix those highlight groups rather than trying to fix it in this one single case

Reply:
I will update the link to DiffText and DiffAdd and we can fix the colorgroup for this in a separate PR?

Reply:
Yup! Sounds great! I'll open a PR to resolve those highlights

Reply:
This is now resolved.